CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Changelog
 
 ### 4.0.0 (Next)
+* [#176](https://github.com/alexa-js/alexa-app/pull/176): Added flexibility to attach alexa-app to express app or router - [@bgjehu](https://github.com/bgjehu).
```

Comment:
@dblock ok, I see now. In https://github.com/alexa-js/alexa-app-server/blob/master/index.js#L122 it shows that the expressApp could be an instance of a router, not necessary an instance of an app. But the param naming is confusing then.
And what would be the reason to bind alexa-app to newly created router in root and bind it to something else thru alexa.express(). Wouldn't it be clearer to users that alexa.express() takes in an express router, bind alexa-app to the router to given endpoint instead of bind to a new router under root and bind the router to given endpoint?